helpers.lib.sh

```diff
@@ -42,6 +42,7 @@ test_go(){
 
 # tests if the cothority exists and is on the correct branch
 test_cothority() {
+    return 0
```

Comment:
Please remove this line, commit, then re-add it but don't commit, instead use `git update-index --assume-unchanged <file>`